### DISCUSSION TITLE
How can I execute command from terminal?

### DISCUSSION THREAD

**Discussion by SoyDiego** [kovidgoyal/kitty#7078] [Q&A] [answered]
Hi @kovidgoyal, first of all, good work with Kitty, it's amazing!!!

Now my problem, I was reading a lot of things but I cannot do it. I want to automatize some tasks but I cannot find the solution yet.
I would like two things:

1. When I execute mi `.sh`, I would like open 3 windows (In my case is opening 4. the current where I executed the command and the other 3)
2. I don't know how can I send `ENTER`. I can write the path but I don't know how can I send keys. I tried a lot of things.

Here you have a gif:

![kitty](https://github.com/kovidgoyal/kitty/assets/19764334/56f0cc49-d45e-4e02-98a7-5046ce0051ae)

I hope someone can help or guide me.

Thanks

**Comment by SoyDiego** [comment]
I was able to make a little progress by doing this. Is it ok? or can we improve it?

```
kitty @ launch sh -c "cd ~/projects/project1 && ng s -o && n && exec zsh"
kitty @ launch sh -c "cd ~/projects/project2 && npm run dev && exec zsh"
```

Thanks!

**Comment by SoyDiego** [comment, marked as answer]
Finally I found the solution.
The first project should be open without `kitty` and the rest with `kitty`.
My final code:

```
cd ~/projects/project1 && ng s -o &
kitty @ launch sh -c "cd ~/projects/project2 && npm run dev"
kitty @ launch sh -c "cd ~/projects/project3 && npm run dev"
```

If there is a better option, let me know :)

**Comment by ad-chaos** [comment]
Could also do `kitten @ launch --copy-env --cwd=~/project npm run dev`

https://sw.kovidgoyal.net/kitty/remote-control/#kitten-launch
